src/codeu/chat/client/maingui/MainGui.java

```diff
@@ -0,0 +1,77 @@
+package codeu.chat.client.maingui;
+
+import java.awt.*;
+import javax.swing.*;
+import javax.swing.border.Border;
+
+import codeu.chat.client.ClientContext;
+import codeu.chat.client.Controller;
+import codeu.chat.client.View;
+import codeu.chat.util.Logger;
+import codeu.chat.common.LoginInputCallback;
+
+// Chat - top-level client application - Java Simple GUI (using Java Swing)
```

Comment:
Standard javadoc formatting for a class or method comment would be to put this in /**, i.e.:
````java
/**
 * Chat - top-level client application...
 */
````